snap/snapcraft.yaml

```diff
@@ -44,40 +44,21 @@ parts:
     source-type: git
     # 'source-tag' and 'python-packages' are automatically generated and managed by
     # tools/update_snapcraft.py, and therefore, should not be manually modified.
-    source-tag: 34.1.0
+    source-tag: 38.0.0
     python-packages:
-    - confluent-kafka==1.8.2
-    - git+https://opendev.org/openstack/barbican-tempest-plugin.git@2.0.0
-    - git+https://opendev.org/openstack/blazar-tempest-plugin.git@0.10.1
-    - git+https://opendev.org/openstack/cinder-tempest-plugin.git@1.8.1
-    - git+https://opendev.org/openstack/cloudkitty-tempest-plugin.git@2.7.1
-    - git+https://opendev.org/openstack/cyborg-tempest-plugin.git@2.1.0
-    - git+https://opendev.org/openstack/designate-tempest-plugin.git@0.17.0
-    - git+https://opendev.org/openstack/ec2api-tempest-plugin.git@1.6.1
-    - git+https://opendev.org/openstack/freezer-tempest-plugin.git@3.0.1
-    - git+https://opendev.org/openstack/glance-tempest-plugin.git@0.6.0
-    - git+https://opendev.org/openstack/heat-tempest-plugin.git@1.7.1
-    - git+https://opendev.org/openstack/ironic-tempest-plugin.git@2.6.1
-    - git+https://opendev.org/openstack/keystone-tempest-plugin.git@0.12.0
-    - git+https://opendev.org/openstack/kuryr-tempest-plugin.git@0.14.1
-    - git+https://opendev.org/openstack/magnum-tempest-plugin.git@2.0.0
-    - git+https://opendev.org/openstack/manila-tempest-plugin.git@1.13.0
-    - git+https://opendev.org/openstack/mistral-tempest-plugin.git@2.1.1
-    - git+https://opendev.org/openstack/monasca-tempest-plugin.git@2.6.0
-    - git+https://opendev.org/openstack/murano-tempest-plugin.git@2.6.0
-    - git+https://opendev.org/openstack/neutron-tempest-plugin.git@2.2.0
-    - git+https://opendev.org/openstack/octavia-tempest-plugin.git@2.3.0
-    - git+https://opendev.org/openstack/oswin-tempest-plugin.git@2.1.0
-    - git+https://opendev.org/openstack/senlin-tempest-plugin.git@1.7.1
-    - git+https://opendev.org/openstack/solum-tempest-plugin.git@2.6.0
-    - git+https://opendev.org/openstack/telemetry-tempest-plugin.git@1.9.0
-    - git+https://opendev.org/openstack/trove-tempest-plugin.git@2.0.1
-    - git+https://opendev.org/openstack/venus-tempest-plugin.git@3.0.0
-    - git+https://opendev.org/openstack/vitrage-tempest-plugin.git@6.1.1
-    - git+https://opendev.org/openstack/watcher-tempest-plugin.git@2.6.0
-    - git+https://opendev.org/openstack/zaqar-tempest-plugin.git@1.6.0
-    - git+https://opendev.org/openstack/zun-tempest-plugin.git@4.7.1
-    - git+https://opendev.org/openinfra/python-tempestconf.git@3.4.0
+    - git+https://opendev.org/openstack/barbican-tempest-plugin.git@4.0.0
+    - git+https://opendev.org/openstack/cinder-tempest-plugin.git@1.12.0
+    - git+https://opendev.org/openstack/designate-tempest-plugin.git@0.22.0
+    - git+https://opendev.org/openstack/glance-tempest-plugin.git@0.8.0
```

Comment:
Looking at this output I thought that the algorithm would check for every plugin and see if there are two releases and take the last element. Why [glance](https://opendev.org/openstack/releases/src/branch/master/deliverables/caracal/glance-tempest-plugin.yaml) appears here and [cloudkitty](https://opendev.org/openstack/releases/src/branch/master/deliverables/caracal/cloudkitty-tempest-plugin.yaml), [cyborg](https://opendev.org/openstack/releases/src/branch/master/deliverables/caracal/cyborg-tempest-plugin.yaml) and etc are excluded? It's not clear to me

Reply:
cloudkitty, cyborg, and others are included in the default exclude list: https://github.com/canonical/snap-tempest-automation/blob/main/excluded-plugins.txt